ps1/ps1/src/twitter/SocialNetwork.java

```diff
@@ -1,134 +0,0 @@
-import java.util.Set;
-import static java.util.stream.Collectors.*;
-import static java.util.Map.Entry.*;
-
-/**
- * SocialNetwork provides methods that operate on a social network.
- * 
- * A social network is represented by a Map<String, Set<String>> where map[A] is
- * the set of people that person A follows on Twitter, and all people are
- * represented by their Twitter usernames. Users can't follow themselves. If A
- * doesn't follow anybody, then map[A] may be the empty set, or A may not even
- * exist as a key in the map; this is true even if A is followed by other people
- * in the network. Twitter usernames are not case sensitive, so "ernie" is the
- * same as "ERNie". A username should appear at most once as a key in the map or
- * in any given map[A] set.
- * 
- * DO NOT change the method signatures and specifications of these methods, but
- * you should implement their method bodies, and you may add new public or
- * private methods or classes if you like.
- */
-public class SocialNetwork {
-
-    /**
-     * Guess who might follow whom, from evidence found in tweets.
-     * 
-     * @param tweets
-     *            a list of tweets providing the evidence, not modified by this
-     *            method.
-     * @return a social network (as defined above) in which Ernie follows Bert
-     *         if and only if there is evidence for it in the given list of
-     *         tweets. One kind of evidence that Ernie follows Bert is if Ernie
-     * @-mentions Bert in a tweet. This must be implemented. Other kinds of
-     *            evidence may be used at the implementor's discretion. All the
-     *            Twitter usernames in the returned social network must be
-     *            either authors or @-mentions in the list of tweets.
-     */
-    public static Map<String, Set<String>> guessFollowsGraph(List<Tweet> tweets) {
-        
-        Map<String, Set<String>> userNetwork = new HashMap<>();
-        
-        // get all mentioned users
-        Set<String> userList = Extract.getMentionedUsers(tweets);
-        
-        // add authors who weren't mentioned to this list
-        // to get total user list
-        for (int i = 0; i < tweets.size(); i++){
-            if (!userList.contains(tweets.get(i).getAuthor().toLowerCase())){
-                userList.add(tweets.get(i).getAuthor().toLowerCase());
-            } else {
-                continue;
-            }
-        }
-        
-        // now cycle through to find tweets authored or @-mentioned
-        for (String user : userList) {
-            
-            user = user.toLowerCase();
-                     
-            // set of users mentioned in own tweets
-            List<Tweet> ownTweets = Filter.writtenBy(tweets, user);
-            Set<String> Network = Extract.getMentionedUsers(ownTweets);
-            
-            // set of authors mentioning user in their tweets
-            // error here: you need to get ALL tweets where they're mentioned,
```

Comment:
is it still the case?

Reply:
yes... I forgot about case sensitivity until I was running my tests. Then I tried to go back and fix but I missed some parts...